docssrc/src/permissions.md

```diff
@@ -117,3 +117,145 @@ Now we declare our command with arguments. We use a `PlayerArgument` and apply t
 > As you can see, there are multiple ways of applying permissions to commands with arguments. In the `/god` command shown above, the permission was applied to the whole command. In the `/kill` command shown above, the permission was applied to the argument.
 >
 > There's not really much difference between the two methods, but I personally would use _argument permissions_ as it has greater control over arguments.
+
+## Child Base Permissions
```

Comment:
After this heading, we need to describe what we're trying to achieve here. (Something along the lines of creating 'child' permissions using `plugin.yml` declaration. Also need to state _why_ you'd want to do something like this (e.g. to fine-tune permissions, to keep things easier to manage in permission-based plugins or something))